modules/simple-payments/simple-payments.css

```diff
@@ -0,0 +1,17 @@
+.jetpack-simple-payments__purchase-message {
+	display: none;
+	color: #fff;
```

Comment:
I'm sorry Takashi. I didn't see your comment last night. It was tacked in this commit https://github.com/Automattic/jetpack/commit/59ea83a9dfe3373ad56fb7d3cc457768f949b3dc of the #7500